### PR TITLE
WRC 4 added

### PR DESCRIPTION
**Pull request by ToRRent1812**
Let's make it run out of the box.

### CODE REVIEW

File: gamefixes-steam/256330.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+def main():
+    #Black screen without it
+    util.protontricks('d3dx9_42')
+    util.protontricks('d3dx9_43')
+    util.protontricks('faudio')
+    util.protontricks('xact')
+
+    # Audio breaks above 60 fps
+    util.set_environment('DXVK_FRAME_RATE', '60')
```
Comment (R1kaB3rN) [inline gamefixes-steam/256330.py:15]: Instead of this fix, a PR should be made for this upstream in DXVK.

File: gamefixes-steam/256330.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    #Black screen without it
+    util.protontricks('d3dx9_42')
+    util.protontricks('d3dx9_43')
+    util.protontricks('faudio')
```
Comment (R1kaB3rN) [inline gamefixes-steam/256330.py:11]: Why does this solve? Using the faudio verb will install downgraded faudio DLLs and use them. Are you sure this is necessary?

File: gamefixes-steam/256330.py (hunk 1 above)
Comment (ToRRent1812) [reply gamefixes-steam/256330.py:15]: audio bug is not linux related, should I still ask DXVK to limit framerate upstream?

File: gamefixes-steam/256330.py (hunk 2 above)
Comment (ToRRent1812) [reply gamefixes-steam/256330.py:11]: Tried without it, looks like it works now, without xact, audio sliders do not work in game. I also forgot to add wmp11 to the script to make background videos work.

File: gamefixes-steam/256330.py (hunk 1 above)
Comment (R1kaB3rN) [reply gamefixes-steam/256330.py:19]: Yes, please make a PR to DXVK as everyone will benefit and you're certain that playing the game above 60+ FPS will break audio. Also, I'm not even too sure we allow fixes to set this variable as this fix would be only one that sets this environment variable in this project.

**Comment by R1kaB3rN** [comment]
I'll merge this ASAP once you make a DXVK PR and remove that environment variable.

See https://github.com/doitsujin/dxvk/blob/master/src/util/config/config.cpp for examples.

**Pull request merged**